hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/impl/SqlClientConnection.java

```diff
@@ -36,27 +33,21 @@
  */
 public class SqlClientConnection implements ReactiveConnection {
 
-	private static final CoreMessageLogger log = CoreLogging.messageLogger("org.hibernate.SQL");
-
 	private static PropertyKind<Long> mySqlLastInsertedId;
 
-	private final boolean showSQL;
-	private final boolean formatSQL;
-	private final boolean highlightSQL;
+	private final SqlStatementLogger sqlStatementLogger;
 	private final boolean usePostgresStyleParameters;
 
 	private final Pool pool;
 	private final SqlConnection connection;
 	private Transaction transaction;
 
 	SqlClientConnection(SqlConnection connection, Pool pool,
```

Comment:
@Sanne This might affect what you where doing for Quarkus

Reply:
It shouldn't, I don't think.

Reply:
The class that in Quarkus extends the SqlClientPool needed some refactoring and these properties were a bit problematic when I talked to him about it. This PR might make things easier.

Anyway, I don't think this is a blocking issue for us. I just wanted the mentioned that this part is changing to him.

Reply:
Yeah, but this is a change that's completely *internal* to `SqlClientPool`.

Reply:
Sure, ok

Reply:
Thanks, yes this is useful to know.